{"uid":"83323a444d41a2d0","name":"test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","fullName":"tests.catalog.options.test_HU11_ES_POST_ProductOption#test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999","historyId":"09c4f63e9a9984c5936c41cb895fd765","time":{"start":1755294159415,"stop":1755294159594,"duration":179},"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTUyOTQwOTgsImV4cCI6MTc1NTI5NzY5OCwicm9sZX...eOW6jzM-pBdCcgPaqxV4rJ61kGswnMCUzkWdTi2YwQ90x8VI5MuTVSGAH1pxXcQ8Hb_RWIs1GBp7I_Of-ddBedMgJ0DxsBgBhaoS7Ssd51Ly4YN4'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755294098314,"stop":1755294098315,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"auth_headers","time":{"start":1755294098315,"stop":1755294098830,"duration":515},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_options","time":{"start":1755294159414,"stop":1755294159414,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","statusTrace":"setup_add_options = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTUyOTQwOTgsImV4cCI6MTc1NTI5NzY5OCwicm9sZX...eOW6jzM-pBdCcgPaqxV4rJ61kGswnMCUzkWdTi2YwQ90x8VI5MuTVSGAH1pxXcQ8Hb_RWIs1GBp7I_Of-ddBedMgJ0DxsBgBhaoS7Ssd51Ly4YN4'}, [])\n\n    @pytest.mark.functional\n    @pytest.mark.boundary\n    @pytest.mark.xfail(reason=\"Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\", run=True)\n    def test_TC227_Verificar_error_al_crear_opcion_con_position_mayor_a_999999999(setup_add_options):\n        headers, _ = setup_add_options\n        payload = generate_options_source_data()\n        payload[\"position\"] = fake.random_int(min=1000000000, max=10000000000)\n        response = SyliusRequest.post(EndpointOptions.options(), headers, payload)\n        log_request_response(EndpointOptions.options(), response, headers, payload)\n>       AssertionStatusCode.assert_status_code_422(response)\n\ntests/catalog/options/test_HU11_ES_POST_ProductOption.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>\n\n    @staticmethod\n    def assert_status_code_422(response):\n>       assert response.status_code == 422\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:59: AssertionError","steps":[],"attachments":[{"uid":"31781320344c560b","name":"log","source":"31781320344c560b.txt","type":"text/plain","size":2192}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"setup_add_options::0","time":{"start":1755294159600,"stop":1755294159600,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"boundary"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.catalog.options"},{"name":"suite","value":"test_HU11_ES_POST_ProductOption"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2377-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.catalog.options.test_HU11_ES_POST_ProductOption"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":5,"passed":0,"unknown":0,"total":5},"items":[{"uid":"93720d030acb58a0","reportUrl":"https://Elv500.github.io/Softsign/30//#testresult/93720d030acb58a0","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755291850629,"stop":1755291850830,"duration":201}},{"uid":"495c6ca4e3fbe1bd","reportUrl":"https://Elv500.github.io/Softsign/29//#testresult/495c6ca4e3fbe1bd","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755291507948,"stop":1755291508131,"duration":183}},{"uid":"b86f7337b87654f1","reportUrl":"https://Elv500.github.io/Softsign/28//#testresult/b86f7337b87654f1","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nrequests.exceptions.JSONDecodeError: Expecting value: line 3 column 1 (char 2)","time":{"start":1755291270204,"stop":1755291270204,"duration":0}},{"uid":"c558ba7d6c4bdaec","reportUrl":"https://Elv500.github.io/Softsign/27//#testresult/c558ba7d6c4bdaec","status":"skipped","statusDetails":"XFAIL Known issue BUG: Error 500 (Internal Server Error) inapropiado al validar campo numérico\n\nAssertionError","time":{"start":1755149134150,"stop":1755149134429,"duration":279}}]},"tags":["boundary","functional"]},"source":"83323a444d41a2d0.json","parameterValues":[]}